com.unity.netcode.gameobjects/Editor/HiddenScriptEditor.cs

```diff
@@ -1,5 +1,4 @@
 using Unity.Netcode.Components;
-using Unity.Netcode.Transports.UNET;
```

Comment:
thanks for the fix but I think this should've been caught by CI — are we not testing any Unity version without UNet? I think we need to add that newer Unity version into our CI pipeline so that we could catch future UNet-related issues earlier @ashwinimurt 

(also, which Unity version you were using @DenninDalke?)

Reply:
> thanks for the fix but I think this should've been caught by CI — are we not testing any Unity version without UNet? I think we need to add that newer Unity version into our CI pipeline so that we could catch future UNet-related issues earlier @ashwinimurt
> 
> (also, which Unity version you were using @DenninDalke?)

@ThusSpokeNomad We do not run any 2022.2 / trunk CI (where Unet is removed) as part of our PR checks. So the PR got merged. The nightlies would have failed anyway. 
Should I add a 2022.2 config to the PR triggers?

Reply:
> thanks for the fix but I think this should've been caught by CI — are we not testing any Unity version without UNet? I think we need to add that newer Unity version into our CI pipeline so that we could catch future UNet-related issues earlier @ashwinimurt
> 
> (also, which Unity version you were using @DenninDalke?)

@ThusSpokeNomad @DenninDalke I believe this is better fix https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/2214
Lets close this PR and merge @ShadauxCat's PR above.

Reply:
AFAIK, 2022.2 is still a pre-release, so I can't really tell whether or not to "absolutely" add it to CI pipeline — but if you think it'd be a low-effort and we don't have huge blockers preventing us from doing it, I do find it beneficial to add 2022.2 (or 2022.x in general) to our CI pipeline.

I also agree that #2214 seems like a more proper fix for this issue with asmdef guarding.

I'll leave CI changes decision up to you @ashwinimurt, whenever you think we should, we will.

Reply:
We have 2022.2 in CI, its not part of PR Triggers, it runs nightly. I can add one of the 2022.2 configs to PR triggers.

Reply:
I was testing it with `2022.2.0b5` I'll close this PR, thanks for the insights!